### ISSUE TITLE
Are you sure it installs ?

### ISSUE THREAD

**Issue by vindarel** [tsikov/vcr#1]
Hi,
this is a welcome package !
To install it I put it in `~/.quicklisp/local-projects/` and `(ql:quickload :vcr)`'ed it, but I get the following error:

![selection_140](https://user-images.githubusercontent.com/3721004/27205412-ef51b570-5230-11e7-8494-1ef40cb2e832.png)

and I don't understand it :/ Any ideas ?

thanks

**Comment by tsikov** [comment]
Hi @vindarel!

I tried to install it on my PC and got a `read error`:

```
;   READ error during COMPILE-FILE:
;
;     Package DRAKMA does not exist.
````

I made a fix for my error, but I am not entirely sure that it will fix your issue too. Can you pull the latest commit and try to load it again? If this doesn't work, can you please try to clone `vcr` inside `~/common-lisp/` and try to load it again with `(ql:quickload :vcr)`.

If this doesn't work I will contact you by email to investigate further.

Thanks!

P.S. Maybe it is an OS specific thing, but shouldn't the directory be `~/quicklisp/local-projects/` instead of `~/.quicklisp/local-projects/`?

**Comment by vindarel** [comment]
It installs and works fine now !

At my first test I may have messed up something. I setted `(setf vcr:*original-fn-symbol* 'dex:get)` and I got a DEXADOR.ERROR:HTTP-REQUEST-BAD-REQUEST. Then I tried with drakma, without setting the original-fn-symbol and I got an CL-SSL:SSL-ERROR-VERIFY. Then by doing again with dexador the presentation trace showed it was using drakma. Eventually I hit an `Unhandled memory fault at #44` (!), but only once then again the cert error. Anyway it's working just fine in the other window, with the same url :)
